src/tools/dev/scripts/bv_support/bv_hdf5.sh

```diff
@@ -489,6 +489,35 @@ EOF
     return 0;
 }
 
+function apply_hdf5_resize_configs_are_equal_patch
+{
+    info "Patching hdf5 1.8.14 for isatty"
```

Comment:
Is this comment correct?

Reply:
Doh! I don't think so. Will fix shortly.

Reply:
```suggestion
    info "Patching hdf5 1.8.14 for resize configs are equal"
```